CHANGELOG.md

```diff
@@ -4,6 +4,64 @@ All notable changes to Zebra are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-beta.4](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-beta.4) - 2022-01-26
+
+Zebra's latest beta improves the networking code and fixes some bugs. A couple of fixed bugs had
+caused Zebra to hang in some situations. Some improvements to the documentation were also included.
+
+### Added
+
+- Add a copy-state zebrad command, which copies blocks between two state services (#3175)
+
+#### Networking
+
+- Add isolated Tor connection API, but don't enable it by default (#3303)
+- Add a test for message broadcast to the right number of peers (#3284)
+
+### Changed
+
+- Update to use Rust edition 2021 (#3332)
+
+#### Networking
+
+- Cache incoming unsolicited address messages, and use them as responses (#3294)
+- Cleanup internal network request handler, fix unused request logging (#3295)
+
+#### Documentation
+
+- Document the consensus rules for Section 3.6 (#3338)
```

Comment:
@upbqdn can you add the section title to this changelog entry?

Reply:
Done, let me know if it's okay.

Reply:
Looks good.